### PR TITLE
backend/entities: add person entity for lecturers etc.

### PR DESCRIPTION
**Pull request by JohaJung**
Addresses the backend part of #33 

---

### Implementation

- [ ] Does this code change accomplish what it is supposed to do?
- [ ] Can this solution be simplified without unreasonable effort
- [ ] Is the code modular enough?
- [ ] Does similar functionality already exist in the codebase? If yes, why isn’t it reused?
- [ ] Single Responsibility Principle: Does any class serve only one purpose? If not, it should probably split up in multiple classes.
- [ ] Interface Segregation: If interface implemented, is it fully implemented?
- [ ] Dependency Injection: Are central object instantiation processes managed by a single context?
- [ ] If any functionality is missing, is it marked with a comment that starts with TODO:?
- [ ] Do you see any potential to improve the performance of the code significantly? 

### Logic Errors and Bugs

- [ ] Can you think of any use case in which the code does not behave as intended?
- [ ] Can you think of any inputs or external events that could break the code?
- [ ] Are edge cases covered by tests?

### Error Handling and Logging

- [ ] Is error handling done the correct way?
- [ ] Should any logging or debugging information be added or removed?
- [ ] Are error messages user-friendly?

### Dependencies (internal and external)

- [ ] Were updates to documentation, configuration, or readme files made as required by this change?
- [ ] Is the API.md up-to-date?
- [ ] Are the enumerations between frontend and backend synchronized?

### Security and Data Privacy

- [ ] Does the code introduce any security vulnerabilities?
- [ ] Is (user) input validated, sanitized, and escaped to prevent security attacks such as cross-site scripting or SQL injection?
- [ ] Is sensitive data like user data handled and stored securely?
- [ ] Is the right encryption used?
- [ ] Is data retrieved from external APIs or libraries checked for security issues?
- [ ] Were any secrets committed?

### Testing and Testability (later)

- [ ] Have automated tests been added, or have related ones been updated to cover the change?
- [ ] Do the existing tests reasonably cover the code change?
- [ ] Are there some test cases, input or edge cases that should be tested in addition?

### Usability and Accessibility

- [ ] Is the proposed solution (UI) accessible (e.g. colorblindness, ...)?
- [ ] Is the API/UI intuitive to use?
- [ ] Was the app tested on another device?

### Readability

- [ ] Is the code easy to understand?
- [ ] Which parts were confusing to you and why?
- [ ] Can the readability of the code be improved by smaller methods/different names?
- [ ] Is the code located in the right file/folder?
- [ ] Do you think certain methods should be restructured to have a more intuitive control flow?
- [ ] Are there redundant or outdated comments?
- [ ] Could some comments convey the message better?
- [ ] Could some comments be removed by making the code itself more readable?

### CODE REVIEW

File: VLA-Backend/src/main/java/de/vlaorgatu/vlabackend/calendar/person/Person.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import lombok.NoArgsConstructor;
+
+/**
+ * Represents a person, that e.g. holds a lecture, in our calendar.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(force = true)
+@Entity
+@Table(name = "personas")
```
Comment (polodasi123) [inline VLA-Backend/src/main/java/de/vlaorgatu/vlabackend/calendar/person/Person.java]: why personas instead of persons or people?

File: VLA-Backend/src/main/java/de/vlaorgatu/vlabackend/calendar/person/Person.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+public class Person {
+    /**
+     * Primary key.
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```
[the remark this answers is not in the document]
Comment (JohaJung) [reply VLA-Backend/src/main/java/de/vlaorgatu/vlabackend/calendar/person/Person.java]: Sadly, this throws an error when using the CrudRepository for creating new instances without specifying an ID.

With generationType auto it is possible and the entity gets assigned a new id.

So, I'll go forward with the default (AUTO) here, it does not even need to be specified as argument (See the first changed file here: https://github.com/TUDa-VLAOrga/VLAOrga/pull/69/changes/2e6838c58ec7118b469bb8a4784850bc1c96db23#diff-a68f2926e023d93a2de7adda485b11868e33df1a0ace377f8b6003690fa761adR23 )

File: VLA-Backend/src/main/java/de/vlaorgatu/vlabackend/calendar/person/Person.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+     * Name of this person.
+     */
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    /**
+     * Personal notes, specifying e.g. equipment preferences of this lecturer.
+     */
+    @Column(name = "note", nullable = false)
+    private String note = "";
```
[the remark this answers is not in the document]
Comment (polodasi123) [reply VLA-Backend/src/main/java/de/vlaorgatu/vlabackend/calendar/person/Person.java]: okay dann lass es so
